src/commands/codepush/lib/codepush-release-command-base.ts

```diff
@@ -103,24 +103,24 @@ export default class CodePushReleaseCommandBase extends AppCommand {
     this.deploymentName = this.specifiedDeploymentName;
 
     if (this.privateKeyPath) {
-      const appInfo = (
-        await out.progress(
-          "Getting app info...",
-          clientRequest<models.AppResponse>((cb) => client.appsOperations.get(this.app.ownerName, this.app.appName, cb))
-        )
-      ).result;
-      const platform = appInfo.platform.toLowerCase();
+      // const appInfo = (
```

Comment:
Can you please add a reference in the readme about the exact location of the commented lines and brief explanation of the reason why we have to do that?